LocalSettings.txt

```diff
@@ -46,3 +46,13 @@ $wgLogos = [
 	]
 ];
 $wgFavicon = "$wgResourceBasePath/favicon.ico";
+
+// PageViewInfo
+$wgPageViewInfoWikimediaDomain = 'en.wikipedia.org';
+
+// GrowthExperiments
+$wgGERestbaseUrl = 'https://en.wikipedia.org/api/rest_v1';
+$wgGENewcomerTasksRemoteApiUrl = 'https://en.wikipedia.org/w/api.php';
+$wgGENewcomerTasksTopicType = 'ores';
+$wgWelcomeSurveyExperimentalGroups['exp2_target_specialpage']['range'] = '0-9';
+$wgGEHomepageMentorsList = 'Mentors';
```

Comment:
I would make this page name more specific so it's less likely to clash with imported data, something like "Project:GrowthExperiments/Mentors"

Reply:
I initially put it in the Project namespace, but that leads to an incredibly ugly (normalized) page title like "Patch_Demo_(616810,6):GrowthExperiments/Mentors". You're right that it does belong in the Project namespace though, so I'll put it there.

Reply:
I think setting the canonical project namespace to "Project" would be sensible.

Reply:
Submitted as a separate PR: #96